src/core/components/HeaderBarMenu/HeaderBarMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import './HeaderBarMenu.css';
+import { IconButton } from '../IconButton';
+import { BurgerMenuIcon } from '../../../assets/icons/BurgerMenuIcon';
+import { ProfileIcon } from '../../../assets/icons/ProfileIcon';
+import { useAppDispatch } from '../../../config/redux/hooks';
+import { callHeaderBlockButtonClicked } from '../../redux/coreSlice';
+import { useState } from 'react';
+
+interface HeaderBarMenuProps {
+  companyName: string;
+}
+
+export enum HEADER_BUTTON_ID {
+  burgerMenuButton = 'burger-menu-button',
+  profileMenuButton = 'profile-menu-button',
+}
+
+export const HeaderBarMenu = ({ companyName }: HeaderBarMenuProps) => {
+  const dispatch = useAppDispatch();
+
+  const [buttonID, setButtonID] = useState('');
```

Comment:
Please remove it. If we need to control the state of the current buttons, it should be processed via a redux slice, as it probably will be connected to a state of some other components (for example, a parent page that is not in the nav bar menu)